test/OpenSpaceToolkit/Astrodynamics/Trajectory/Segment.test.cpp

```diff
@@ -202,6 +203,27 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Segment, SegmentSolution_Comput
     }
 }
 
+TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Segment, SegmentSolution_ComputeAccelerations)
+{
+    {
+        const Segment::Solution segmentSolution =
+            Segment::Solution(defaultName_, defaultDynamics_, {defaultState_}, true, Segment::Type::Coast);
+
+        Array<MatrixXd> accelerations = segmentSolution.computeAccelerations();
+
+        std::cout << accelerations << std::endl;
+    }
+
+    {
+        const Segment::Solution segmentSolution =
+            Segment::Solution(defaultName_, defaultDynamics_, {defaultState_}, true, Segment::Type::Coast);
+
+        Array<MatrixXd> accelerations = segmentSolution.computeAccelerations();
+
+        std::cout << accelerations << std::endl;
+    }
+}
```

Comment:
This test looks like the same exact one as the previous one? Also don't you want to actually check that the values match the default ones and not print stuff to console? (maybe this is still a draft in which case I imagine you were planning on doing that)

Reply:
> maybe this is still a draft in which case I imagine you were planning on doing that

Precisely ;)